src/shared/string_op.c

```diff
@@ -910,36 +910,24 @@ char * wstr_unescape_json(const char * string) {
     return output;
 }
 
-/* Check if the specified string is alphanumeric */
-bool is_string_alphanumeric(const char* string) {
-    bool ret_val = FALSE;
-    if (NULL != string) {
-        bool is_alpha = true;
-        int i = 0;
-        while (is_alpha && string[i] != '\0') {
-            is_alpha &= 0 == isalnum(string[i]) ? false : true;
-            ++i;
-        }
-        ret_val = is_alpha;
-    }
-    
-    return ret_val;
-}
-
-/* Convert string to time_t */
-time_t string_to_time_t(char* string, const time_t default_value) {
-    time_t ret_val = default_value;
-    if (NULL != string && w_str_is_number(string)) {
-        if (sizeof(time_t) == sizeof(int)) {
-            ret_val = atoi(string);
-        } else if (sizeof(time_t) == sizeof(long)) {
-            ret_val = atol(string);
-        } else if (sizeof(time_t) == sizeof(long long)) {
-            ret_val = atoll(string);
-        } else {
-            mwarn("Unrecognized time_t value type.");
-        }
+// Lowercase a string
+
+char * w_tolower_str(const char *string) {
```

Comment:
this is adapting to how is the master branch:
https://github.com/wazuh/wazuh/blob/ff962364251c1fa38468bc750c8431ced9254b84/src/shared/string_op.c#L915